firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/common/TestUtils.java

```diff
@@ -33,4 +35,15 @@ public static void writeStringToFile(String s, File f) throws IOException {
       }
     }
   }
+
+  public static byte[] gzipToBytes(byte[] compressed, int numBytes) {
```

Comment:
Can we call this `gzipToRawBytes`, or `inflateGzip` or somesuch?

Reply:
Also, using a `ByteArrayOutputStream` would allow you to not have to pass in the number of bytes of the uncompressed output.